tests/data/udf/shift_28_bits_one_day/udf.sql

```diff
@@ -3,13 +3,11 @@
 Shift input bits one day left and drop any bits beyond 28 days.
```

Comment:
Why are some of these UDFs moved under `tests/data/udf` rather than `udf/`?

Reply:
I see now that these are duplicated in both spots, and that the `tests/data` versions are used for tests. I'm a bit worried about this duplication. I could imagine there being some confusion for someone browsing the repo looking for an implementation of one of these duplicated functions. Or if we need to make a modification to one of them, is it expected that we do so in both places?

Could we prefix all of these names with `example_` or something along those lines to make the separation more clear?

Reply:
I prefixed them with `test_` which is in line with some of the other test data that we already have.